Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Curtis Vogt <curtis.vogt@gmail.com>"]
 version = "1.19.0"
 
 [deps]
+Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
```

Comment:
Need to update the `docs/Manifest.toml`

Reply:
It works without a Manifest there. It seems reasonable to me but again I am not familiar with Documenter.

Reply:
Let's drop it. It is still needed for local dev but this means less to maintain